client/lib/signup/hint-data.js

```diff
@@ -1,673 +1,677 @@
+	'Freelance marketer': i18n.translate( 'Freelance marketer' ),
+	'Freelance services': i18n.translate( 'Freelance services' ),
+	'Freelance writer': i18n.translate( 'Freelance writer' ),
+	'Funeral services': i18n.translate( 'Funeral services' ),
+	Furniture: i18n.translate( 'Furniture' ),
+	Games: i18n.translate( 'Games' ),
+	Gaming: i18n.translate( 'Gaming' ),
+	Garments: i18n.translate( 'Garments' ),
+	Genealogy: i18n.translate( 'Genealogy' ),
+	Glamour: i18n.translate( 'Glamour' ),
+	Gloves: i18n.translate( 'Gloves' ),
+	God: i18n.translate( 'God' ),
+	Golf: i18n.translate( 'Golf' ),
+	Gospel: i18n.translate( 'Gospel' ),
+	'Government and politics': i18n.translate( 'Government and politics' ),
+	'Graphic design': i18n.translate( 'Graphic design' ),
+	'Graphic novels': i18n.translate( 'Graphic novels' ),
+	Graphics: i18n.translate( 'Graphics' ),
+	'Green technology': i18n.translate( 'Green technology' ),
+	Groceries: i18n.translate( 'Groceries' ),
+	'Guitar teacher': i18n.translate( 'Guitar teacher' ),
+	Guns: i18n.translate( 'Guns' ),
+	Gym: i18n.translate( 'Gym' ),
+	HIV: i18n.translate( 'HIV' ),
+	HVAC: i18n.translate( 'HVAC' ),
+	Hacking: i18n.translate( 'Hacking' ),
+	'Hair salon': i18n.translate( 'Hair salon' ),
+	'Hair styles': i18n.translate( 'Hair styles' ),
+	Happiness: i18n.translate( 'Happiness' ),
+	Hardware: i18n.translate( 'Hardware' ),
+	Healing: i18n.translate( 'Healing' ),
+	Health: i18n.translate( 'Health' ),
+	'Health and beauty': i18n.translate( 'Health and beauty' ),
+	'Health and fitness': i18n.translate( 'Health and fitness' ),
+	Healthcare: i18n.translate( 'Healthcare' ),
+	'Heat transfer vinyl': i18n.translate( 'Heat transfer vinyl' ),
+	Hinduism: i18n.translate( 'Hinduism' ),
+	History: i18n.translate( 'History' ),
+	Hobbies: i18n.translate( 'Hobbies' ),
+	Holiday: i18n.translate( 'Holiday' ),
+	'Holiday rentals': i18n.translate( 'Holiday rentals' ),
+	Holidays: i18n.translate( 'Holidays' ),
+	'Holistic healing': i18n.translate( 'Holistic healing' ),
+	Home: i18n.translate( 'Home' ),
+	'Home decor': i18n.translate( 'Home decor' ),
+	'Home improvement': i18n.translate( 'Home improvement' ),
+	'Home inspection': i18n.translate( 'Home inspection' ),
+	'Home renovation': i18n.translate( 'Home renovation' ),
+	Homework: i18n.translate( 'Homework' ),
+	Horses: i18n.translate( 'Horses' ),
+	Hospital: i18n.translate( 'Hospital' ),
+	Hospitality: i18n.translate( 'Hospitality' ),
+	Hotel: i18n.translate( 'Hotel' ),
+	'House cleaning': i18n.translate( 'House cleaning' ),
+	Housekeeping: i18n.translate( 'Housekeeping' ),
+	'How to': i18n.translate( 'How to' ),
+	'Human rights issues': i18n.translate( 'Human rights issues' ),
+	'Humanitarian aid': i18n.translate( 'Humanitarian aid' ),
+	Hunting: i18n.translate( 'Hunting' ),
+	IT: i18n.translate( 'IT' ),
+	'IVF treatment': i18n.translate( 'IVF treatment' ),
+	Ideas: i18n.translate( 'Ideas' ),
+	Illustration: i18n.translate( 'Illustration' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Turn off illustrations' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).